### ISSUE TITLE
If unspecified, starting appearance should be random

### ISSUE THREAD

**Issue by I-am-Erk** [CleverRaven/Cataclysm-DDA#60049]
### Is your feature request related to a problem? Please describe.

If you don't select your appearance traits at start, your character winds up without appearance traits at all, which is actually sort of an error state, albeit not one I think we should be trapping for.

### Solution you would like.

If appearance traits are not selected, a player should use randomized appearance from the trait groups I created many years ago:

See [appearance_trait_groups.json](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/npcs/appearance_trait_groups.json) for examples. I suggest just using `appearance_demographics`.

### Describe alternatives you have considered.

- It is worth considering making "no appearance traits" an error state that causes random `appearance_demographics` stuff to be selected. However there are some potential problems here eg. if a player has a mutant appearance that isn't correctly recognized as an appearance trait and then gets overridden.
- It would be worth considering allowing the player to, instead of selecting specific appearances, select a trait group from a list to focus the randomization down... but I think that adds too much complexity both to the feature and to the chargen window myself.

### Additional context

Honestly I thought we fixed this a few years ago.

**Comment by Fris0uman** [comment]
What about selecting random appearance trait when you open character creation? So instead of a blank slate you always have some cosmetic traits that you can edit or ignore.This way we don't need to fgure out if a character has some appearance selected or not at the end chargen

**Comment by I-am-Erk** [comment]
That would work, yeah.

**Comment by PatrikLundell** [comment]
Yes, appearance traits ought to work like e.g. blood type, so you'd always have one randomly selected as a starting point (as suggested, selecting them from groups of consistent set of traits initially would make sense). It ought to be possible to override traits through mutations (eye color doesn't make sense if you don't have any eyes at all, for instance), and mutations may also enable new values that aren't available normally (unnatural eye color or "skin" color, for instance), as well as new traits (fur patterns, horn shape, etc.).
However, ensuring the standard appearance traits have a value is a good start.
